config/settings/development.yml

```diff
@@ -1,6 +1,9 @@
 allow_impersonation: true
 relative_url_root: '/useradmin'
 
+# How many days away does expiration need to be before we start to worry?
+expires_soon_in_days: 30
```

Comment:
Any reason not to make this the default in `config/settings.yml` and omit it here and in `config/settings/test.yml`?